### ISSUE TITLE
UpdateNotificationDialog: not accessible for users of screen readers.

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#9942]
### Bug description

UpdateNotificationDialog currently uses an wxHtmlWindow, which is not accessible for users of screen readers. There already exists the class HtmlWindow which is accessible.

### Steps to reproduce

Read the code

### Expected behavior

Dialog is accessible for users of screen readers

### Actual behavior

Dialog is not accessible for users of screen readers

### Audacity Version

Audacity 3.7.6

### Operating system

Windows

**Comment by kryksyh** [comment]
@DavidBailes thank you!
